conf/modules/relativelocalizationfilter.xml

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="relativelocalizationfilter" dir="relativelocalizationfilter">
```

Comment:
indent here is not correct, pprz uses 2 spaces not tabs. Please refer to the wiki for this. I also recommend that if you are using an IDE to use paparazzi.sublime-project or paparazzi_code_profile_eclipse.xml